### PR TITLE
Fix model

### PR DESCRIPTION
**Pull request by lvnhmd**

### CODE REVIEW

File: pages/establishment/update/views/index.jsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ class Repeat extends Component {
 
               <Fieldset
                 {...this.props}
-                model={mapKeys(authorisations.filter(auth => auth.type !== undefined), (val, key) => `authorisation-${authorisation.type}-${key}-${authorisation.id}`)}
+                model={mapKeys(authorisations.filter(auth => auth.type !== undefined)[index], (val, key) => `authorisation-${authorisation.type}-${key}-${authorisation.id}`)}
```
Comment (lennym) [inline pages/establishment/update/views/index.jsx]: Taking the same index after filtering the array is likely to break by returning an unexpected entry. I don't understand what the underlying issue is, but my best guess is that you want to retreive the entry at the index, and then cehck its type property.

Comment (lvnhmd) [reply pages/establishment/update/views/index.jsx]: obsolete, it turned out to be another bug

File: pages/establishment/update/views/index.jsx [hunk 2]
```diff
@@ -74,8 +74,7 @@ class Repeat extends Component {
         authorisations: authorisations.map((authorisation, i) => {
           if (i === index) {
             return {
-              ...authorisation,
-              ...model
+              ...authorisation
```
Comment (lennym) [inline pages/establishment/update/views/index.jsx]: This isn't doing anything at all to the state. Either `i === index` and it returns the authorisation, or it doesn't and it returns... the authorisation.

There's no update.

Comment (lvnhmd) [reply pages/establishment/update/views/index.jsx]: Removed onFieldsetChange - we do not need it.

**Pull request merged**